### ISSUE TITLE
two more type errors found (although they look identical to me)

### ISSUE THREAD

**Issue by Iexhu** [ififfy/flipflip#207]
Hi,

Flipflip wont show main user interface, i am on the latest version.
Copy pasted from developer tools;

renderer.bundle.js:343 TypeError: Cannot read property 'url' of undefined
    at Kb (renderer.bundle.js:604)
    at i (renderer.bundle.js:604)
    at py.componentDidMount (renderer.bundle.js:604)
    at qo (renderer.bundle.js:343)
    at Vo (renderer.bundle.js:343)
    at renderer.bundle.js:343
    at Object.t.unstable_runWithPriority (renderer.bundle.js:351)
    at Ps (renderer.bundle.js:343)
    at As (renderer.bundle.js:343)
    at Ts (renderer.bundle.js:343)
    at xs (renderer.bundle.js:343)
    at Zo (renderer.bundle.js:343)
    at Ds (renderer.bundle.js:343)
    at zs (renderer.bundle.js:343)
    at Us.render (renderer.bundle.js:343)
    at renderer.bundle.js:343
    at Ms (renderer.bundle.js:343)
    at Hs (renderer.bundle.js:343)
    at Object.render (renderer.bundle.js:343)
    at Module.<anonymous> (renderer.bundle.js:604)
    at n (renderer.bundle.js:1)
    at renderer.bundle.js:1
    at renderer.bundle.js:1
po @ renderer.bundle.js:343
renderer.bundle.js:343 Uncaught TypeError: Cannot read property 'url' of undefined
    at Kb (renderer.bundle.js:604)
    at i (renderer.bundle.js:604)
    at py.componentDidMount (renderer.bundle.js:604)
    at qo (renderer.bundle.js:343)
    at Vo (renderer.bundle.js:343)
    at renderer.bundle.js:343
    at Object.t.unstable_runWithPriority (renderer.bundle.js:351)
    at Ps (renderer.bundle.js:343)
    at As (renderer.bundle.js:343)
    at Ts (renderer.bundle.js:343)
    at xs (renderer.bundle.js:343)
    at Zo (renderer.bundle.js:343)
    at Ds (renderer.bundle.js:343)
    at zs (renderer.bundle.js:343)
    at Us.render (renderer.bundle.js:343)
    at renderer.bundle.js:343
    at Ms (renderer.bundle.js:343)
    at Hs (renderer.bundle.js:343)
    at Object.render (renderer.bundle.js:343)
    at Module.<anonymous> (renderer.bundle.js:604)
    at n (renderer.bundle.js:1)
    at renderer.bundle.js:1
    at renderer.bundle.js:1

Keep up the great work. Thanks for your time!

**Comment by StarrHelixx** [comment]
Hey @Iexhu , thanks for the error report. 

Do you happen to remember what you were doing (what form it was trying to load) when you got this error? Unfortunately, it's hard for me to track down the exact issue just based on the stacktrace, but I will try to reproduce where I see `.url` being used.

**Comment by Iexhu** [comment]
Hi StarHelixx,

Lets see if i can remember. I was experimenting with grid 2X2 playing both random vids and pics. 2x same clips location 2x same pic location, so only two sources if that makes sense. Dont ask me specific pic and clip settings, nothing fancy but random timings for both iirc. Neither really fast either.  

On another note. Almost always after a while the screen goes white, alas i never checked for errors there since i just closed and re-opend the program. I will try to remember to look once i can get in again that is.  

Hope this helps, Thanks!

**Comment by StarrHelixx** [comment]
>  I was experimenting with grid 2X2...

🤔 Hmmm... ok, well I did find a bug when you try to play a scene with no sources in a grid. I got a different error message than you, but I'm pretty sure that was the issue. This is now fixed on `dev`.

> Almost always after a while the screen goes white...

I'm not sure exactly what would be causing this. If you _do_ happen to remember to check the console (and it will actually show up), let me know what you find -- I'd really appreciate it! 😁
